README.rst

```diff
@@ -116,7 +113,7 @@ version using ::
 With other Python distributions the program can be upgraded to
 the latest version as follows ::
 
-     easy_install --upgrade diffpy.pdfmorph
+     conda update diffpy.pdfmorph
```

Comment:
This strikes me as not correct based on the text just above.  I suggest that we simply delete  lines starting "with other python distributions......" and just leave instructions how to update the software using conda.